cypress/e2e/organizations/user-integration-organization.cy.js

```diff
@@ -70,7 +70,7 @@ describe('ui-organizations: EDI convention in Organization Integration', () => {
     // Found and edit created organization
     Organizations.searchByParameters('Name', organization.name);
     Organizations.checkSearchResults(organization);
-    Organizations.chooseOrganizationFromList(organization);
+    Organizations.selectOrganization(organization.name);
```

Comment:
you used waiter on 81 line please add a comment to explain why you need it there